src/commands/database/list.mjs

```diff
@@ -29,23 +29,24 @@ function pickOutputFields(databases, argv) {
 }
 
 async function listDatabasesWithAccountAPI(argv) {
-  const { pageSize, database, json, color } = argv;
+  const { pageSize, database, color } = argv;
   const accountClient = new FaunaAccountClient();
   const response = await accountClient.listDatabases({
     pageSize,
     path: database,
   });
   const output = pickOutputFields(response.results, argv);
 
-  if (json) {
-    container.resolve("logger").stdout(JSON.stringify(output));
-  } else {
-    container.resolve("logger").stdout(formatObjectForShell(output, { color }));
-  }
+  container.resolve("logger").stdout(
+    await colorize(output, {
+      format: JSON_FORMAT,
+      color: color,
+    }),
+  );
```

Comment:
This means that the json option is ignored for the list database list command, we'll just always return JSON, is that right?

Reply:
Yeah. There's not another format right now.